pkg/authorization/registry/rolebinding/policybased/virtual_storage.go

```diff
@@ -248,13 +252,23 @@ func (m *VirtualStorage) updateRoleBinding(ctx kapi.Context, name string, objInf
 	return updatedRoleBinding, false, nil
 }
 
+// modifyingRoleForEscalation tries to use the CachedRuleResolver if available to avoid expensive checks
+func (m *VirtualStorage) modifyingRoleForEscalation(binding authorizationinterfaces.RoleBinding) (authorizationinterfaces.Role, error) {
```

Comment:
I understand the method, but not the name.  Looks like I made the name initially, but it sucks.